geometric/run_json.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+import json
+import geometric
+
+def get_qc_schema_traj(qc_schema_input, progress):
+    qc_schema_traj = []
+
+    return qc_schema_traj
+
+def parse_json_input(in_json):
+    """
+    Parse an input json file into options, example:
+    {
+        "schema_name": "qc_schema_optimization_input",
+        "schema_version", 1,
+        "geometric_options": {
+            "coordsys": "tric",
+            "conv": 1.e-7
```

Comment:
The input parameters set here will be directly passed to `geometric.optimize.OptParams()`